scripts/checkout_merge_commit.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+
+# Add `master` branch to the update list.
+# Otherwise CircleCI will give us a cached one.
+FETCH_REFS="+master:master"
```

Comment:
We need this for 2 purposes.

1. Purging merged recipes. ( xref: https://github.com/conda-forge/staged-recipes/pull/1467 )
2. To error out PR builds that have merge conflicts.